components/Avatar/Avatar.css

```diff
@@ -0,0 +1,135 @@
+@import url('https://fonts.googleapis.com/css2?family=Dongle&family=Open+Sans:ital@1&family=Source+Sans+Pro:ital,wght@0,700;1,400&display=swap');
+@import url("../../style.css");
+@import url("../../color-palette/color.css");
+
+*, body{
+    margin:0;
+    padding:0;
+}
+.tab-header{
```

Comment:
Try using rems for values greater than 10px.

Reply:
yea i have implemented it . Thanks for reviewing